third_party/2and3/click/types.pyi

```diff
@@ -71,7 +71,13 @@ class FloatParamType(ParamType):
 
 
 class FloatRange(FloatParamType):
-    ...
+    def __init__(
+        self,
+        min: float = None,
+        max: float = None,
+        clamp: bool = False,
+    ) -> None:
```

Comment:
Please use `...` for default arguments. This is why flake8 fails. Also, since `min` and `max` accept `None`, their type should by `Optional[float]`.

Reply:
done :)